bluepyopt/ephys/protocols.py

```diff
@@ -192,7 +193,20 @@ def _run_func(self, cell_model, param_values, sim=None):
             cell_model.freeze(param_values)
             cell_model.instantiate(sim=sim)
 
-            self.instantiate(sim=sim, icell=cell_model.icell)
+            if isinstance(cell_model, models.LFPyCellModel):
+                LFPyCell = cell_model.LFPyCell
+            else:
+                LFPyCell = None
+
+            self.instantiate(
+                sim=sim, icell=cell_model.icell, LFPyCell=LFPyCell
```

Comment:
I'm not sure I understand the necessity. Can't the LFPyCell be store inside the cell_model object? (I.e. why does this logic have to be here?)